schema/author.graphql

```diff
@@ -0,0 +1,42 @@
+type Author {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    createdAt: Int
+    updatedAt: Int
+    deletedAt: Int
+}
+
+input AuthorFilter {
```

Comment:
We are not using this anywhere?

Reply:
Not at this moment, I think we should delete it. What do you suggest?

Reply:
If you think this will be used in future then you can keep it